app/controllers/application_controller.rb

```diff
@@ -370,6 +370,6 @@ def respond_with_json(reponse)
   end
 
   def hide_page?(subject)
-    subject.viewer?(current_user) && current_user.mutual_allies?(subject.user)
```

Comment:
Not sure why this logic was changed to this... This is the opposite of what we want lol.